### PR TITLE
Implement Like Logic in Frontend

### LINKED ISSUE #28: Likes UI and Integeration with API
API : POST - **/like_unlike/**

### PR DESCRIPTION
**Pull request by aitikgupta**
Fixes https://github.com/Open-Sourced-Olaf/SocioMark/issues/28

### CODE REVIEW

File: frontend/src/Components/Post/Post.js [hunk 1]
```diff
@@ -17,6 +18,46 @@ export default function Post({ post_initializer }) {
     setPost(new_post);
   };
 
+  const handleLike = (post_id) => {
+    console.log(post_id);
```
Comment (bodhisha) [inline frontend/src/Components/Post/Post.js]: I see a number of console logs.. :p

**Review by bodhisha — APPROVED** [review]
Just need to remove the log statements.. rest seems good.. 👍 🎉

File: frontend/src/Components/Post/Post.js (hunk 1 above)
Comment (aitikgupta) [reply frontend/src/Components/Post/Post.js]: Uhh, I should've looked twice. :')

Comment (aitikgupta) [reply frontend/src/Components/Post/Post.js]: Fixed 👍🏼

**Pull request merged**